src/commands/download_file.ts

```diff
@@ -0,0 +1,43 @@
+import { EID } from 'ardrive-core-js';
+import { cliArDriveAnonymousFactory, cliArDriveFactory } from '../index';
+import { CLICommand, ParametersHelper } from '../CLICommand';
+import { CLIAction } from '../CLICommand/action';
+import { DrivePrivacyParameters, FileIdParameter, LocalFilePathDownloadParameter } from '../parameter_declarations';
+import { getOutputFilePathAndName } from '../utils';
+import { join } from 'path';
+
+new CLICommand({
+	name: 'download-file',
+	parameters: [FileIdParameter, LocalFilePathDownloadParameter, ...DrivePrivacyParameters],
+	action: new CLIAction(async (options) => {
+		const parameters = new ParametersHelper(options);
+		const fileId = parameters.getRequiredParameterValue(FileIdParameter, EID);
+		const destOutputPath = parameters.getParameterValue(LocalFilePathDownloadParameter) || './';
```

Comment:
Windows won't be happy with that extra `/`
```suggestion
		const destOutputPath = parameters.getParameterValue(LocalFilePathDownloadParameter) || '.';
```